From ebffab3ca6ed7d77c3cd7b4d920c2053320c1d83 Mon Sep 17 00:00:00 2001
From: WofWca <wofwca@protonmail.com>
Date: Fri, 11 Sep 2026 19:59:12 +0400
Subject: [PATCH] improvement: CVAR defaults: -random +linear

A gib looked more like an explosion.
That might be good for missiles but not for shotgun or railgun.
---
 code/cgame/cg_cvar.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/code/cgame/cg_cvar.h b/code/cgame/cg_cvar.h
index ab076883e..a2a987a1d 100644
--- a/code/cgame/cg_cvar.h
+++ b/code/cgame/cg_cvar.h
@@ -85,12 +85,12 @@ CG_CVAR( cg_gibsPiecesFromKnockback, "cg_gibsPiecesFromKnockback", "1.1", CVAR_A
 	"Q: \"What sense does it make to have more gibs? "
 	"Did the player got bigger from taking more damage??\":\n"
 	"A: No, they simlpy got split into more pieces." )
-CG_CVAR( cg_gibsLinearVelocityFromKnockback, "cg_gibsLinearVelocityFromKnockback", "0.75", 0,
+CG_CVAR( cg_gibsLinearVelocityFromKnockback, "cg_gibsLinearVelocityFromKnockback", "1", 0,
 	"Scale how fast the gibs move away from damage (knockback).\n"
 	"This sounds similar to `cg_gibsInheritPlayerVelocity`, "
 	"but this var doesn't scale the velocity that the player already had "
 	"when they received the damage." )
-CG_CVAR( cg_gibsRandomVelocityFromKnockback, "cg_gibsRandomVelocityFromKnockback", "0.4", CVAR_ARCHIVE,
+CG_CVAR( cg_gibsRandomVelocityFromKnockback, "cg_gibsRandomVelocityFromKnockback", "0.3", CVAR_ARCHIVE,
 	"Scale how much random velocity (spread) the gibs gain "
 	"from damage (knockback)." )
 CG_CVAR( cg_gibsVerticalVelocityFromKnockback, "cg_gibsVerticalVelocityFromKnockback", "0.2", CVAR_ARCHIVE,
-- 
2.55.0

